### DISCUSSION TITLE
编译报错 怎么解决下？

### DISCUSSION THREAD

**Discussion by zhaocy1217** [ejoy/ant#101] [General]
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\include\stdatomic.h(15): fatal error C1189: #error:  <stdatomic.h> is not yet supported when compiling as C, but this is planned for a future release.

**Comment by cloudwu** [comment]
提问题请在 discussions

**Comment by wk34u** [comment]
@zhaocy1217 尝试升级下 Visual Studio?

**Comment by cloudfreexiao** [comment]
看起来是 vs 版本太低
